docs/workflows/manage-content.rst

```diff
@@ -31,7 +31,18 @@ related to manifests because of that. In pulp, it is required to specify
 a digest of a manifest in order to create a tag for the corresponding
 image.
 
-``http POST http://localhost:24817/pulp/api/v3/docker/tag/ repository=${REPOSITORY_HREF} tag=${TAG_NAME} digest=${MANIFEST_DIGEST}``
+Here is provided an example on how to properly tag an image. First, a
+digest of an eligible manifest is selected. Then, a custom tag is applied
+to the corresponding manifest.
+
+.. literalinclude:: ../_scripts/image_tagging.sh
+   :language: bash
+
+After the tagging, a new distribution can be created. This allows clients
```

Comment:
A new distribution can be created to include newly created Tag. This will allow clients to....